db/migrate/20220603192619_add_user_ref_to_likes.rb

```diff
@@ -0,0 +1,5 @@
+class AddUserRefToLikes < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :likes, :user, null: false, foreign_key: true
```

Comment:
Please add `author_id` instead of `user_id` as a foreign key to the `likes` table as depicted here in this [ERD](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_app_erd.png).

Reply:
Hello @SajjadAhmad14 , thank you for the review. From my understanding the author_id in the tables that the ERD ask for are the user that created them, so the user_id is the same thing as the author_id since there's no authors table. Do you want me to create a new table named authors? Or how should I go about fixing this issue?